lib/features/email/presentation/extensions/presentation_email_extension.dart

```diff
@@ -0,0 +1,18 @@
+
+import 'package:model/email/presentation_email.dart';
+import 'package:model/extensions/list_email_header_extension.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/smime_signature_status.dart';
+
+extension PresentationEmailExtension on PresentationEmail {
+
+  SMimeSignatureStatus get sMimeStatus {
+    final status = emailHeader?.toSet().sMimeStatus;
+    if (status == 'Good signature') {
```

Comment:
please define constant for 'Good signature' and 'Bad signature' in SMimeConstant

Reply:
done